server/src/graphql/resolvers/event/event.ts

```diff
@@ -32,14 +32,34 @@ const eventResolvers: {
           events.map<Event>(({ name, location, timeAndDate, type, _id }) => ({
             name,
             location,
-            timeAndDate: timeAndDate.toString(),
+            timeAndDate: new Date(timeAndDate).getTime(),
```

Comment:
is it possible timeAndDate will be undefind?

Reply:
i added default value to `timeAndDate`